<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Singko</string>
  </property>
  <widget class="QPushButton" name="head">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="usr_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>300</width>
     <height>400</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 5pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="contacts">
    <attribute name="title">
     <string>Contacts</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="groups">
    <attribute name="title">
     <string>Groups</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Recent</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;</string>
   </property>
   <property name="text">
    <string>Singko</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_pushbutton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Ã—</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>head</zorder>
  <zorder>usr_name</zorder>
  <zorder>label</zorder>
  <zorder>close_pushbutton</zorder>
  <zorder>logo</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>head</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
